zkevm-circuits/src/witness/tx.rs

```diff
@@ -866,7 +901,7 @@ impl From<MockTransaction> for Transaction {
             call_data: mock_tx.input.to_vec(),
             call_data_length: mock_tx.input.len(),
             call_data_gas_cost: tx_data_gas_cost(&mock_tx.input),
-            access_list_gas_cost: tx_access_list_gas_cost(&Some(mock_tx.access_list)),
+            access_list_gas_cost: tx_access_list_gas_cost(&access_list),
```

Comment:
you won't need this as well then

Reply:
Removed in the same commit [6880378](https://github.com/scroll-tech/zkevm-circuits/pull/1045/commits/68803783b1447b8fefe2c1fabd626beae51b8a83).